system/views/admin/datamanager/batchEditField.cfm

```diff
@@ -0,0 +1,35 @@
+<cfscript>
+
+	param name="args.cancelAction"  type="string"  default=event.buildAdminLink( linkTo="datamanager.object", querystring='id=#rc.object#' );
+	object              = rc.object       ?: "";
+	renderObject        = rc.renderObject ?: "";
+	renderSwitch        = rc.renderSwitch ?: "";
+	recordLabel         = rc.pickField    ?: "";
+	objectTitleSingular = translateResource( uri="preside-objects.#object#:title.singular", defaultValue=object ?: "" );
+	editRecordTitle     = translateResource( uri="cms:datamanager.bulkEdit.title", data=[ LCase( objectTitleSingular ),recordLabel,ListLen(rc.id) ] );
+	saveButton          = translateResource( uri="cms:datamanager.savechanges.btn", data=[ LCase( objectTitleSingular ) ] );
+	prc.pageIcon        = "pencil";
+	prc.pageTitle       = editRecordTitle;
+</cfscript>
+<cfoutput>
+	<form class="form-horizontal" method="post" data-dirty-form="protect" action="#event.buildAdminLink( linkTo='datamanager.batchEditAction')#">
+			<input type="hidden" name="sourceIds"   value="#rc.id#">
```

Comment:
Watch the indentation here, there is a double indent here where there should only be one level.